libvcx/src/api_lib/api_handle/credential_def.rs

```diff
@@ -122,14 +139,17 @@ pub fn check_is_published(handle: u32) -> VcxResult<bool> {
     })
 }
 
-pub fn rotate_rev_reg_def(handle: u32, revocation_details: &str, new_tails_url: &str) -> VcxResult<String> {
+// todo: add 'publish revocation primitives' method
```

Comment:
The name of the PR and the description suggest this should be part of the scope (and I agree)

Reply:
Added `publish_revocation_primitives` but currently unused and this asks for question whether we should separate `publish` as `publish_cred_def` and `publish_revocation_primitives`, or keep `publish` as a "smart" method which would be used for both (the current state).
I'd keep `publish` for libvcx, eg. keep the api as is.

Reply:
If we keep `publish` and FFI API as is, adding unused `publish_revocation_primitives` to handles layer doesn't make much sense - let's remove it

Reply:
removed after all